src/plugins/base.eth/handlers/Registrar.ts

```diff
@@ -17,33 +17,19 @@ const {
 
 export default function () {
   // support NameRegisteredWithRecord for BaseRegistrar as it used by Base's RegistrarControllers
-  ponder.on(pluginNamespace("BaseRegistrar:NameRegisteredWithRecord"), async ({ context, event }) =>
-    handleNameRegistered({ context, event }),
-  );
+  ponder.on(pluginNamespace("BaseRegistrar:NameRegisteredWithRecord"), handleNameRegistered);
 
-  ponder.on(pluginNamespace("BaseRegistrar:NameRegistered"), async ({ context, event }) => {
-    await upsertAccount(context, event.args.owner);
-    // Base has 'preminted' names via Registrar#registerOnly, which explicitly
-    // does not update the Registry. This breaks a subgraph assumption, as it
-    // expects a domain to exist (via Registry:NewOwner) before any
-    // Registrar:NameRegistered events. We insert the domain entity here,
-    // allowing the base indexer to progress.
-    await context.db.insert(schema.domain).values({
-      id: makeSubnodeNamehash(ownedSubnameNode, tokenIdToLabel(event.args.id)),
-      ownerId: event.args.owner,
-      createdAt: event.block.timestamp,
-    });
```

Comment:
This insert caused an issue for [the initial Transfer handler](https://github.com/namehash/ensnode/pull/40/files#diff-5cc50dbf0d6ef2a9d7148562b56e141dc7440cddfe373882a411e9b41b023a1fR30). But now, we will always ensure a new domain is inserted into database on the initial transfer event (when `from` arg is the zero address).